sqlalchemy_redshift/dialect.py

```diff
@@ -25,14 +25,18 @@
 from .compat import string_types
 
 try:
-    from alembic.ddl import postgresql
+    import alembic
 except ImportError:
     pass
 else:
+    from alembic.ddl import postgresql
```

Comment:
I made this change because it makes it more clear that we're simply checking if Alembic is installed. Plus, we need another import anyway to check `alembic.__version__`.